packages/opentelemetry-instrumentation-watsonx/README.md

```diff
@@ -0,0 +1,21 @@
+# OpenTelemetry IBM Watsonx Instrumentation
+
+This library allows tracing VertexAI prompts and completions sent with the official [IBM Watson Machine Learning library](https://ibm.github.io/watson-machine-learning-sdk/).
+
+## Installation
+
+```bash
+pip install opentelemetry-instrumentation-watsonx
```

Comment:
@nirga does traceloop will help release opentelemetry-instrumentation-watsonx and let customer install via pip install?

Reply:
Yes :) We just need to update some stuff in the CI similar to these 2 together
https://github.com/traceloop/openllmetry/pull/285
https://github.com/traceloop/openllmetry/pull/316

Reply:
Awesome, thanks @nirga !